src/test/java/com/wikia/webdriver/common/core/annotations/Driver.java

```diff
@@ -0,0 +1,13 @@
+package com.wikia.webdriver.common.core.annotations;
+
+/**
+ * Created by Ludwik on 2015-02-19. Added LANGUAGE2 Japanese. By P. Archbold on 2015-07-23
+ */
+public class Driver {
+
+  public static final String Chrome = "CHROME";
+  public static final String Firefox = "FF";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)